### PR TITLE
Add functionality for uploading profile picture

### LINKED ISSUE #86: Implement profile picture functionality

### PR DESCRIPTION
**Pull request by fymbc**
WIP

Closes #86

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/UploadCommand.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+    public static final String MESSAGE_UPLOAD_PERSON_SUCCESS = "Uploaded Profile Picture for Person: %1$s";
+    public static final String MESSAGE_UPLOAD_CANCELLED = "Cancelled profile picture upload for Person: %1$s";
+    public static final String MESSAGE_UPLOAD_ERROR = "Error getting profile picture. "
+            + "Did you ensure it is a .png file?";
+    private final Index targetIndex;
+
+    /**
+     * Default constructor for a UploadCommand object
+     * @param targetIndex Index to be deleted
```
Comment (AlphaPyke) [inline src/main/java/seedu/address/logic/commands/UploadCommand.java:43]: is this "deleted" because the target will be replaced?

Comment (fymbc) [reply src/main/java/seedu/address/logic/commands/UploadCommand.java:43]: Nope, typo on my end. Will change to be "uploaded"

File: src/main/java/seedu/address/logic/commands/UploadCommand.java [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        //
+        //            public boolean accept(File f) {
+        //                if (f.isDirectory()) {
+        //                    return true;
+        //                } else {
+        //                    String filename = f.getName().toLowerCase();
+        //                    return filename.endsWith(".png");
+        //                }
+        //            }
+        //        });
```
Comment (Justin-Aw46) [inline src/main/java/seedu/address/logic/commands/UploadCommand.java:78]: Do you need this part? If not could delete it since its commented out.

**Comment by AlphaPyke** [comment]
LGTM, I will take over from here.

**Pull request merged**